config.yaml

```diff
@@ -19,7 +19,7 @@ url: 'https://pool.burstcoin.ro'      # mainnet pool
 #url: 'http://localhost:6876'         # solo mining testnet
 
 hdd_reader_thread_count: 0            # default 0 (=auto: number of disks)
-hdd_use_direct_io: true               # default true
+hdd_use_direct_io: true               # default true, use false on not PC
```

Comment:
"use false for remote/network access", is that what you mean?

Reply:
No, jonnyFm said that "little ones" do not support it. I do not know why it not supports..maybe it is related to USB connection and ARM proccesors?